http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -777,6 +779,9 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       raw"""galaxy.ingress.hosts[0]=${k8sProxyHost}""",
       raw"""galaxy.configs.galaxy\.yml.galaxy.single_user=${userEmail.value}""",
       raw"""galaxy.configs.galaxy\.yml.galaxy.admin_users=${userEmail.value}""",
+      // TODO add this when Galaxy version is upgraded
+      //raw"""galaxy.configs.file_sources_conf\.yml[0].workspace=${workspaceName}""",
```

Comment:
I'm coordinating with Alex to get the latest Galaxy packaged, at which point we can uncomment this.